src/Circle.fs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Fabulous.XamarinForms.Maps
+
+open System.Collections.Generic
+open System.Runtime.CompilerServices
+open Fabulous
+open Fabulous.StackAllocatedCollections
+open Fabulous.StackAllocatedCollections.StackList
+open Fabulous.XamarinForms
+open Xamarin.Forms.Maps
+
+type ICircle =
+    inherit IMapElement
+
+module Circle =
+    let WidgetKey = Widgets.register<Circle> ()
+
+    let Center = Attributes.defineBindableWithEquality<Position> Circle.CenterProperty
+
+    let Radius = Attributes.defineBindableWithEquality<Distance> Circle.RadiusProperty
+
+    let FillColor = Attributes.defineBindableAppThemeColor Circle.FillColorProperty
+
+[<AutoOpen>]
+module CircleBuilders =
+
+    type Fabulous.XamarinForms.View with
+
+        static member inline Circle<'msg>(center: Position, radius: Distance) =
```

Comment:
Would that create some conflicts with the Circle shape of XF 🤔 
Maybe we should isolate 3rd party widgets into separate View namespaces?

eg. `type Fabulous.XamarinForms.Maps.View`

Usage
```
open Fabulous.XamarinForms
open Fabulous.XamarinForms.Maps

open type Fabulous.XamarinForms.View
open type Fabulous.XamarinForms.Maps.View

VStack() {
    Circle()

    Map()
        .mapElements() {
            Circle(...)
        }
}
```

Reply:
Yeah this can be problematic . We can maybe use ```CircleElement or CircleMarker or CircleShape```  ?

Reply:
Actually I think  isolating 3rd party widgets into separate View namespaces will about the conflicting names.